src/readii/negative_controls.py

```diff
@@ -94,6 +106,7 @@ def makeRandomRoi(
     baseImage: sitk.Image, 
     baseROI: sitk.Image, 
     roiLabel: Optional[int] = None,
```

Comment:
is there a test case for when `roiLabel` is None? just to make sure it works

Reply:
There is not, I will add one.